build.xml

```diff
@@ -1035,4 +1039,11 @@
       </classpath>
     </eclipse>
   </target>
+
+  <target name="check-forbidden-apis" depends="compile">
+    <fa:forbiddenapis internalRuntimeForbidden="true" classpathref="classpath" dir="${build.dir}">
+      <bundledsignatures name="jdk-unsafe-${javac.version}"/>
```

Comment:
yes I think we should make full use of the plugin e.g. add more checks from the available options. If you wish to split these in to separate items and merge them gradually then I would be happy to merge. Please say what you want to do.